clients/spark/core/src/main/scala/io/treeverse/gc/CommittedAddressLister.scala

```diff
@@ -22,14 +23,14 @@ class NaiveCommittedAddressLister extends CommittedAddressLister {
                                          )
     var df = LakeFSContext.newDF(spark, params)
     df = df
+      // TODO (optional): push down a filter to the input format, to filter out absolute addresses!
+      .filter(
+        (col("address_type") === AddressType.RELATIVE.name) ||
+          // Backwards compatability with entries prior to address_type
+          (col("address_type") === AddressType.BY_PREFIX_DEPRECATED.name &&
+            col("address_type").contains("://"))
```

Comment:
Need to check that if you split by `://` you get at least 2 parts, and that the first one doesn't contain `/` or `:`.